src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -3393,7 +3393,7 @@ static void Test()
                 .True("Test();")
                 .True("Microsoft.CodeAnalysis.Runtime.Instrumentation.FlushPayload();")
                 .True(@"Console.WriteLine(""Test"");");
-            checker.Method(4, 1)
+            checker.Method(5, 1)
```

Comment:
>checker.Method(5, 1) [](http://example.com/codeflow?start=12&length=20)

Do we understand what method is added here?

Reply:
We didn't use to generate a parameterless constructor for the entry-point type (which is a reference type). Now we do.